pepper-apis/src/main/java/org/broadinstitute/ddp/route/PostMedicalProviderRoute.java

```diff
@@ -118,4 +92,42 @@ public PostMedicalProviderResponsePayload handle(
         );
     }
 
+    private int insertMedicalProvider(
+            Handle handle,
+            MedicalProviderDto medicalProviderDto
+    ) {
+        return handle.attach(MedicalProviderDao.class).insert(medicalProviderDto);
+    }
+
+    private void logMedicalProviderCreationFailure(String participantGuid, String studyGuid) {
+        LOG.warn(
+                "The number of created medical providers created by user {}"
+                + " in the study {} is not equal to 1",
+                participantGuid, studyGuid
+        );
+    }
+
+    private void logMedicalProviderCreationSuccess(String participantGuid, String studyGuid, String medicalProviderGuid) {
+        LOG.info(
+                "The user {} successfully created a medical provider {} in the study {}",
+                participantGuid, medicalProviderGuid, studyGuid
+        );
+    }
+
+    private void queueDataForExport(Handle handle, long userId, long umbrellaStudyId) {
+        handle.attach(DataExportDao.class).queueDataSync(userId, umbrellaStudyId);
+    }
```

Comment:
Some methods with a descriptive name. I believe their use is warranted even if they are just 1-liners.

Reply:
These "methods" are private and only used once in this file... I don't see the benefit of pulling them out?